clang_tools/main.py

```diff
@@ -38,13 +38,21 @@ def parse_args(args: List[str] = None) -> argparse.Namespace:
         help="Force overwriting the symlink to the installed binary. This will only "
         "overwrite an existing symlink.",
     )
+    parser.add_argument(
+        "-b",
+        "--no-progress-bar",
+        action="store_true",
+        help="Do not display a progress bar for downloads.",
```

Comment:
does `--no-progress-bar` needs a default value, like "False"

Reply:
`default=False` is implied false because of `action="store_true"